packages/viewer/src/components.d.ts

```diff
@@ -4,424 +4,345 @@
-  interface VertexViewerToolbarGroup {
-    direction: ViewerToolbarGroupDirection;
-  }
+    interface SvgIcon {
+    }
+    interface VertexViewer {
+        /**
+          * Enables or disables the default mouse and touch interactions provided by the viewer. Enabled by default.
+         */
+        "cameraControls": boolean;
+        /**
+          * The Client ID associated with your Vertex Application.
+         */
+        "clientId"?: string;
+        /**
+          * An object or JSON encoded string that defines configuration settings for the viewer.
+         */
+        "config"?: Config | string;
+        /**
+          * Sets the default environment for the viewer. This setting is used for auto-configuring network hosts.  Use the `config` property for manually setting hosts.
+          * @see Viewer.config
+         */
+        "configEnv": Environment;
+        "getBaseInteractionHandler": () => Promise<BaseInteractionHandler | undefined>;
+        "getFrame": () => Promise<Frame.Frame | undefined>;
+        "getInteractionHandlers": () => Promise<InteractionHandler[]>;
+        "getJwt": () => Promise<string | undefined>;
+        /**
+          * Enables or disables the default keyboard shortcut interactions provided by the viewer. Enabled by default, requires `cameraControls` being enabled.
+         */
+        "keyboardControls": boolean;
+        /**
+          * Loads the given scene into the viewer and return a `Promise` that resolves when the scene has been loaded. The specified scene is provided as a URN in the following format:   * `urn:vertexvis:scene:<sceneid>`
+          * @param urn The URN of the resource to load.
+         */
+        "load": (urn: string) => Promise<void>;
+        /**
+          * Internal API.
+          * @private
+         */
+        "registerCommand": <R, T>(id: string, factory: CommandFactory<R>, thisArg?: T | undefined) => Promise<Disposable>;
+        /**
+          * Registers and initializes an interaction handler with the viewer. Returns a `Disposable` that should be used to deregister the interaction handler.  `InteractionHandler`s are used to build custom mouse and touch interactions for the viewer. Use `<vertex-viewer camera-controls="false" />` to disable the default camera controls provided by the viewer.
+          * @example ``` class CustomInteractionHandler extends InteractionHandler {   private element: HTMLElement;   private api: InteractionApi;    public dispose(): void {     this.element.removeEventListener('click', this.handleElementClick);   }    public initialize(element: HTMLElement, api: InteractionApi): void {     this.api = api;     this.element = element;     this.element.addEventListener('click', this.handleElementClick);   }    private handleElementClick = (event: MouseEvent) => {     api.tap({ x: event.clientX, y: event.clientY });   } }  const viewer = document.querySelector("vertex-viewer"); viewer.registerInteractionHandler(new CustomInteractionHandler); ```
```

Comment:
we probably want to add running the script that reformats this file to whichever script generates/updates it, is that `build`? maybe add to `postbuild`?